.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+bundle.js
```

Comment:
Put bundle.js inside react-client/dist/ ?

Reply:
@sethshill thanks so much for reviewing 😀. bundle.js is created inside client/dist. I think for .gitignore if we put the file name without location it can still recognize the file and ignores it during commit. Please let me know if you notice its not working